test/Tabs.spec.js

```diff
@@ -0,0 +1,60 @@
+/* eslint react/no-multi-comp: 0, react/prop-types: 0 */
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import { mount } from 'enzyme';
+import { Nav, NavItem, NavLink, TabContent, TabPane } from 'reactstrap';
+import classnames from 'classnames';
+// Not sure if this is correct but didn't want to repeat a whole bunch of code.
+import TabsExample from '../docs/lib/examples/Tabs';
+
+describe('Tabs', () => {
+  it('should render', () => {
+    let tab1 = mount(<TabsExample />);
+    expect(tab1.find('.nav .nav-tabs').length).toBe(1);
+    expect(tab1.find('.nav .nav-item').length).toBe(2);
+    expect(tab1.find('.tab-content').length).toBe(1);
+    expect(tab1.find('.tab-pane').length).toBe(2);
+  });
+  it('should have tab1 as active', () => {
+    let tab1 = mount(<TabsExample />);
+    expect(tab1.find('.nav .nav-item .nav-link').at(0).hasClass('active')).toBe(true);
+    expect(tab1.find('.nav .nav-item .nav-link').at(1).hasClass('active')).toBe(false);
+    expect(tab1.find('.tab-content .tab-pane').at(0).hasClass('active')).toBe(true);
+  });
+  it('should switch to tab2 as active when clicked', () => {
+    let tab1 = mount(<TabsExample />);
+    expect(tab1.find('.nav .nav-item .nav-link').at(0).hasClass('active')).toBe(true);
+    expect(tab1.find('.nav .nav-item .nav-link').at(1).hasClass('active')).toBe(false);
+    expect(tab1.find('.tab-content .tab-pane').at(0).hasClass('active')).toBe(true);
+    tab1.find('.nav .nav-item .nav-link').at(1).simulate('click');
+    expect(tab1.find('.nav .nav-item .nav-link').at(0).hasClass('active')).toBe(false);
+    expect(tab1.find('.nav .nav-item .nav-link').at(1).hasClass('active')).toBe(true);
+    expect(tab1.find('.tab-content .tab-pane').at(1).hasClass('active')).toBe(true);
+    tab1.find('.nav .nav-item .nav-link').at(0).simulate('click');
+  });
+  it('should show no active tabs if active tab id is unknown', () => {
+    let tab1 = mount(<TabsExample />);
+    const instance = tab1.instance();
+    expect(instance instanceof TabsExample).toBe(true);
```

Comment:
The TabExample is a wrapper around the Tab. The aim in this test is to have a click initiated from the wrapper that toggles the TabContent (in this case an invalid case). The clicking and managing ActiveTab is not in the scope of the Tab component. The test still focuses on testing the Tab and not the TabsExample though.